src/common/snippets/include/snippets/utils/params_dumper.hpp

```diff
@@ -0,0 +1,65 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#pragma once
+#ifdef SNIPPETS_DEBUG_CAPS
+
+#include "openvino/util/file_util.hpp"
+#include "snippets/lowered/linear_ir.hpp"
+#include "snippets/lowered/pass/serialize_control_flow.hpp"
+#include "snippets/lowered/pass/serialize_data_flow.hpp"
+#include "snippets/utils/debug_caps_config.hpp"
+
+namespace ov {
+namespace snippets {
+
+class ParamsDump {
+public:
+    explicit ParamsDump(const ov::snippets::lowered::ExpressionPtr& expr,
+                        const DebugCapsConfig& debug_config,
+                        std::string params)
+        : expr(expr),
+          debug_config(debug_config),
+          params(std::move(params)) {
+        dump("_in");
+    }
+    ~ParamsDump() {
+        dump("_out");
+    }
```

Comment:
1. We don't need to print the parameters twice -- in constructor and destructor -- once is enough.
2. From my perspective, a separate class in this case looks like an overkill. Since the feature is solely `BrgemmBlocking` related, we could just extract env variable value from `linear_ir.get_config().debug_config` right in the pass, and place all the printing logic there

Reply:
Done